doc/source/whatsnew/v1.0.2.rst

```diff
@@ -25,7 +25,10 @@ Fixed regressions
 Bug fixes
 ~~~~~~~~~
 
--
+**IO**
+
+- Bug in :class:`HDFStore` that caused it to set to ``int64`` the dtype of a ``datetime64`` column when reading a DataFrame in Python 3 from fixed format written in Python 2 (:issue:`31750`)
```

Comment:
Can you move note to v1.1.0? I don't think this is a 0.25 -> 1.0 regression (correct me if mistaken)

Reply:
done